app/views/find/v2/results/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="app-filter-layout">
+  <div class="app-filter-layout__filter">
+    <div class="app-filter__header">
+      <h2 class="govuk-heading-m">Filters</h2>
+
+      <div class="app-filter__content">
+      </div>
+    </div>
+  </div>
+  <div class="app-filter-layout__content">
+    <ul class="app-search-results">
+      <% @results.each do |course| %>
+        <li class="app-search-results__item" data-qa="course">
+          <%= govuk_summary_card(
+            classes: ["course-summary-card"],
+            title: govuk_link_to(
+              find_course_path(
+                provider_code: course.provider_code,
+                course_code: course.course_code
```

Comment:
On the original implementation we are passing the search params to the show page. I added on the checklist of the outcome card and decided not to pass yet. Until we have the filters implemented at least.